label-application-rule-config-service-impl/src/main/java/org/hypertrace/label/application/rule/config/service/LabelApplicationRuleConfigServiceImpl.java

```diff
@@ -0,0 +1,164 @@
+package org.hypertrace.label.application.rule.config.service;
+
+import com.google.common.util.concurrent.Striped;
+import com.google.protobuf.Duration;
+import io.grpc.Channel;
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.grpc.stub.StreamObserver;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+import org.hypertrace.label.application.rule.config.service.v1.CreateLabelApplicationRuleRequest;
+import org.hypertrace.label.application.rule.config.service.v1.CreateLabelApplicationRuleResponse;
+import org.hypertrace.label.application.rule.config.service.v1.DeleteLabelApplicationRuleRequest;
+import org.hypertrace.label.application.rule.config.service.v1.DeleteLabelApplicationRuleResponse;
+import org.hypertrace.label.application.rule.config.service.v1.GetLabelApplicationRuleRequest;
+import org.hypertrace.label.application.rule.config.service.v1.GetLabelApplicationRuleResponse;
+import org.hypertrace.label.application.rule.config.service.v1.GetLabelApplicationRulesRequest;
+import org.hypertrace.label.application.rule.config.service.v1.GetLabelApplicationRulesResponse;
+import org.hypertrace.label.application.rule.config.service.v1.LabelApplicationRule;
+import org.hypertrace.label.application.rule.config.service.v1.LabelApplicationRuleConfigServiceGrpc;
+import org.hypertrace.label.application.rule.config.service.v1.UpdateLabelApplicationRuleRequest;
+import org.hypertrace.label.application.rule.config.service.v1.UpdateLabelApplicationRuleResponse;
+
+public class LabelApplicationRuleConfigServiceImpl
+    extends LabelApplicationRuleConfigServiceGrpc.LabelApplicationRuleConfigServiceImplBase {
+  private static final int LABEL_LOCK_STRIPE_COUNT = 1000;
```

Comment:
They are not used currently. Plz remove them.

Reply:
okay.